ci-operator/step-registry/mco/conf/day2/pintargetrelease/mco-conf-day2-pintargetrelease-commands.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export TARGET="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
```

Comment:
@sergiordlr suggest to add a validation for the required env var, if not found, exit. this is important if we add this step to existing upgrade chain. because the dependent jobs don't have MCO env var

Reply:
Done